src/server/routes/controllers/Middlewares/checkSession.js

```diff
@@ -0,0 +1,12 @@
+const passport = require('passport')
+const strategyUtils = require('../../../passport/strategyUtils')
+
+function userAuthentication(req, res, next) {
+	if(req.isAuthenticated())
+		next()
+	else
+		passport.authenticate(strategyUtils.STRATEGY_NAME, 
```

Comment:
Declare `strategyUtils.strategyHandlers(req, res, next).loginStrategyHandler` as separated const.
Also declare `passport.authenticate(strategyUtils.STRATEGY_NAME, strategyUtils.strategyHandlers(req, res, next).loginStrategyHandler)` as separated const before invoking it.

Reply:
ok